client/src/views/Login.tsx

```diff
@@ -1,8 +1,67 @@
+import React, { useState } from "react";
+import { useAuthStore } from "../store/authStore";
+import InputField from "../components/InputField";
+import FormError from "../components/FormError";
+
 export default function Login() {
+  const { login } = useAuthStore();
+  const [formData, setFormData] = useState({
+    email: "",
+    password: "",
+  });
+  const [error, setError] = useState<string | null>(null);
+  const [success, setSuccess] = useState<string | null>(null);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setFormData({
+      ...formData,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    setError(null);
+    setSuccess(null);
+    try {
+      await login(formData.email, formData.password);
+      setSuccess("Login successful!");
+    } catch (err) {
+      setError("Login failed. Please check your credentials.");
+    }
+  };
+
   return (
-    <>
-      <div>Login</div>
-      <div>{import.meta.env.VITE_API_URL}</div>
-    </>
+    <div className="flex items-center justify-center min-h-screen bg-indigo-900">
+      <div className="bg-indigo-100 p-8 rounded-lg shadow-lg w-full max-w-md">
+        <h2 className="text-2xl font-bold text-center text-indigo-900 mb-6">
+          Login
+        </h2>
+        <form onSubmit={handleSubmit}>
+          <InputField
+            label="Email"
+            name="email"
+            type="email"
+            value={formData.email}
+            onChange={handleChange}
+          />
+          <InputField
+            label="Password"
+            name="password"
+            type="password"
+            value={formData.password}
+            onChange={handleChange}
+          />
+          {success && <p className="text-green-500">{success}</p>}
```

Comment:
why you create a component succes like how did you do for errors. you can use the component for both sides if you pass a more variable `isError`